stream-chat-android-client/build.gradle

```diff
@@ -35,6 +35,9 @@ android {
             consumerProguardFiles 'consumer-proguard-rules.pro'
         }
     }
+
+    resourcePrefix 'stream_'
```

Comment:
I didn't know it was integrated on the Gradle Plugin, I though it was a Lint Rule we needed to write.
Cool!!!